app/controllers/admin/document_collection_group_document_search_controller.rb

```diff
@@ -18,31 +18,18 @@ def search
 
   def add_by_title
     flash.now[:alert] = "Please enter a search query" if params[:title] && params[:title].empty?
-    @editions = filter.editions if params[:title].present?
+    if params[:title].present?
+      results = Edition.published.with_title_containing(params[:title].strip)
+      @editions = results
```

Comment:
Just out of interest, did you give this a go without calling page in the controller? I'm still curious about whether we actually need to make the pagination call ourselves or whether we can pass it to that Kaminari pagination helper and it'll automatically do it

Reply:
i have not! can definitely give it a go

Reply:
tested - does not work - we need to do pages and params according to the kaminari gem i believe